Capture and display all mail with MailHog in development

louim opened the pull request:
This is a preliminary PR covering issue #298. There is a couple of things I'd like to address / get feedback on before considering it ready to be merged, but it's working great in its current form. See comments inline on the code.

louim commented on group_vars/development:
```diff
@@ -1,4 +1,6 @@
 mysql_root_password: devpw
+mailhog_binary_url: https://github.com/mailhog/MailHog/releases/download/v0.1.7/MailHog_linux_amd64
```
This will not be needed once https://github.com/geerlingguy/ansible-role-mailhog/pull/6 is merged in.

louim commented on group_vars/development:
```diff
@@ -30,6 +32,24 @@ wordpress_sites:
+  - type: dport_limit
+    dport: [ssh]
+    seconds: 300
+    hits: 20
```
This block is mostly a copy of the one in the `all` settings. Because it is not possible to append to a list in Ansible its not possible to only add the rule to open port `8025`, I have to override the whole list.

another option would be to put `ferm_enabled: false` for development by default. what do you guys think?

swalkinshaw replied:
Let's just disable it in development. This goes a little beyond dev/prod parity.

louim commented on requirements.yml:
```diff
@@ -7,3 +7,6 @@
 - name: swapfile
   src: kamaln7.swapfile
+- name: mailhog
+  src: https://github.com/Berdir/ansible-role-mailhog
```
I'm using the fork of the pull request until https://github.com/geerlingguy/ansible-role-mailhog/pull/2 in merged, because the role install ssmtp by default and we do it too, so it would conflict.

retlehs replied:
looks like it got merged thanks to you! :+1:

louim commented:
Thanks to the speedy merge on the Mailhog Ansible role, the PR is now updated and ready to go. I added some information on the readme about how to access Mailhog in development. Feel free to find better wording to explain it! :+1:

swalkinshaw commented:
Tested and working great. Want to squash?

louim commented:
@swalkinshaw done!
